.github/workflows/build.yaml

```diff
@@ -54,6 +49,10 @@ jobs:
         env:
           KUBECONFIG: /home/runner/.kube/config
           STARBOARD_CLI_LOG_LEVEL: "0"
+      - name: Upload code coverage
+        uses: codecov/codecov-action@v1
+        with:
+          files: ./coverage.txt,./itest/coverage.txt
```

Comment:
I'm just curious about how they are merged. If you pass two coverage files, are they merged automatically?

Reply:
Yes. As explained in https://docs.codecov.io/docs/merging-reports we can cat report files to combine them and merge the coverage report

Reply:
Thanks.